Source/WebKit/WebProcess/GPU/media/cocoa/VideoLayerRemoteCocoa.mm

```diff
@@ -103,7 +103,7 @@ - (void)setVideoGravity:(WebCore::MediaPlayerEnums::VideoGravity)videoGravity
 
 - (bool)resizePreservingGravity
 {
-    auto* player = self.mediaPlayerPrivateRemote;
+    RefPtr<WebKit::MediaPlayerPrivateRemote> player = self.mediaPlayerPrivateRemote;
```

Comment:
Will `RefPtr player = ...` work here?

Reply:
Not on my machine, it says that it is not able to infer the template type, I am not sure why. Maybe ObjC?